src/main/java/Duke.java

```diff
@@ -1,10 +1,166 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.time.LocalDate;
+
 public class Duke {
+
+    static ArrayList<Task> list = new ArrayList<Task>();
```

Comment:
```suggestion
    static ArrayList<Task> taskList = new ArrayList<Task>();
```

Reply:
Would adding task will help to make the code more readable to users?

Reply:
Agree with @goalfix's comment here, `taskList` is commonly used by others as well :)